API.md

```diff
@@ -0,0 +1,289 @@
+This is the documentation support for the REST API feature.
+
+Route | Method | Description
+-------- | --------------- | ----------
+/**authors** | GET | Search for available authors.
+/**posts**/`:post_id`/**authors** | GET |  Get all authors from a post
+/**posts**/`:post_id`/**authors** | PUT |  Appends a new author to a post
+/**posts**/`:post_id`/**authors**/`:author_id` | DELETE |  Removes an author from a post
+/**guests** | GET |  Lists of filters all the currently available guests 
```

Comment:
This should be "Lists OR filters current available guests". It's a typo. Going to update it the docs.